pandas/core/generic.py

```diff
@@ -5684,7 +5693,7 @@ def sample(
         obj_len = self.shape[axis]
 
         # Process random_state argument
-        rs = com.random_state(random_state)
+        rs = random_state(random_state)
```

Comment:
now the `random_state` argument from this function shadows the `random_state` import

Might have to change the `pandas.core.common` import back to `from pandas.core import common`, but add the `noqa` command to that line to silence the flake8 warning of the missing `as com` alias